### ISSUE TITLE
0.1.0 broken

### ISSUE THREAD

**Issue by xuwei-k** [non/antimirov#1]
```
Welcome to Scala 2.12.11 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_201).
Type in expressions for evaluation. Or try :help.

scala> antimirov.Parser.parse("a")
java.io.FileNotFoundException: /Users/erik/w/antimirov/core/.jvm/target/scala-2.12/scoverage-data/scoverage.measurements.2630 (No such file or directory)
  at java.io.FileOutputStream.open0(Native Method)
  at java.io.FileOutputStream.open(FileOutputStream.java:270)
  at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
  at java.io.FileWriter.<init>(FileWriter.java:107)
  at scoverage.Invoker$.$anonfun$invoked$1(Invoker.scala:55)
  at scala.collection.mutable.HashMap.getOrElseUpdate(HashMap.scala:86)
  at scoverage.Invoker$.invoked(Invoker.scala:55)
  at antimirov.Parser$.<init>(Parser.scala:41)
  at antimirov.Parser$.<clinit>(Parser.scala)
  ... 36 elided
```

**Comment by non** [comment]
Oh jeez, I guess I published with coverage machinery? Ugh, will republish 0.1.1.

**Comment by non** [comment]
Just published 0.2.0 to Sonatype. Thanks for trying this out and catching that!
